usr.sbin/pkg/config.c

```diff
@@ -500,7 +500,11 @@ config_init(const char *requested_repo)
 		c[REPOS_DIR].list = malloc(sizeof(*c[REPOS_DIR].list));
 		STAILQ_INIT(c[REPOS_DIR].list);
 		cv = malloc(sizeof(struct config_value));
+#ifdef COMPAT_64BIT
+		cv->value = strdup("/etc/pkg64");
```

Comment:
Besides that, we should update paths in:
* https://github.com/CTSRD-CHERI/pkg/blob/1.17.5-cheriabi/libpkg/pkg_config.c:
  * `/etc/pkg/`;
  * `/var/db/pkg`;
  * `/var/cache/pkg`;
  * `/usr/ports` (that's an interesting one).
* https://github.com/CTSRD-CHERI/pkg/blob/1.17.5-cheriabi/src/main.c:
  *  `/usr/sbin/pkg`.

I also wonder what should we do about paths in pkg.conf.sample and manual pages. We could create separate pkg.conf.sample files and mention both paths in the manual pages.

Reply:
I think the manages can be updated with sed. It looked like the bootstrap one could be.